stablehlo/reference/Ops.cpp

```diff
@@ -869,50 +893,39 @@ Tensor evalGatherOp(const Tensor &operand, const Tensor &startIndices,
                     ShapedType resultType) {
   Tensor result(resultType);
   Axes batchDims;
-  for (auto i = 0; i < result.getRank(); ++i)
-    if (!llvm::is_contained(offsetDims, i)) batchDims.push_back(i);
+  for (auto d : result.getAxes())
```

Comment:
Renamed some local variables like the `i` here and below to be consistent with how they are called in the spec.